### ISSUE TITLE
Consider restraining talent tree tooltips to available screen real estate

### ISSUE THREAD

**Issue by roth-michael** [foundryvtt/crucible#1533]
This would probably be pretty tricky given how we render the talent tree, but might be considered an accessibility issue: One should be able to increase the font size (or zoom in) and not have longer talents' descriptions go off-screen.

The issue title is maybe too focused on a possible solution than the general issue, but wasn't sure how to word it succinctly otherwise.
